src/SDDPoptimize.jl

```diff
@@ -86,21 +92,26 @@ function run_SDDP!(model::SPModel,
         noise_scenarios = simulate_scenarios(model.noises, param.forwardPassNumber)
 
         # Forward pass
-        costs, stockTrajectories, _ = forward_simulations(model,
+        costs, stockTrajectories, _, callSolver = forward_simulations(model,
                             param,
                             V,
                             problems,
                             noise_scenarios)
 
+        #Update the number of call
+        countCallSolver = countCallSolver + callSolver
```

Comment:
What do you think to increment like this: 

countCallSolver += callSolver

I personally consider that this notation is clearer, but you may have a different position.